posts/freeing-our-maintainers.md

```diff
@@ -0,0 +1,56 @@
+_TL;DR; We're spending a few months redesigning our volunteering model and giving our key volunteers a break from the work of reviewing community contributions.
+If you use Exercism purely to learn or mentor, there’s nothing here you need to know (although please read if you’re interested!). If you are a track maintainer, wish to contribute to Exercism, or wish to report a bug/problem, then consider this essential reading 🙂_
+
+---
+
+Over the last 6 months we’ve spent a lot of time exploring the future of Exercism, imagining what it would look like for every language track to be the best it could be. We’re incredibly proud of what we have built so far. The 85,000 testimonials that have been given speak to the amazing work our community has done in building our language tracks and mentoring so many students through them. Most importantly, we believe we're just scratching the surface of what is possible. We have huge ideas, hopes, and excitement for all that Exercism can be.  But to achieve them, we first need to resolve some fundamental problems that linger under the surface.
```

Comment:
> and many have left as a result

I wonder if that sounds a bit too much like an exodus? We also have had maintainers sticking it out with us

Reply:
We have, but a larger proportion have left. I think this is honest.

Reply:
Okay!